libgit/repo.go

```diff
@@ -185,7 +147,7 @@ func getOrCreateRepoAndID(
 	ctx context.Context, config libkbfs.Config, tlfHandle *libkbfs.TlfHandle,
 	repoName string, uniqID string, createOnly bool) (*libfs.FS, ID, error) {
 	if !checkValidRepoName(repoName, config) {
-		return nil, NullID, errors.WithStack(InvalidRepoNameError{repoName})
+		return nil, NullID, errors.WithStack(libkb.InvalidRepoNameError{Name: repoName})
```

Comment:
Note that the client changes we're vendoring in include an `ExportToStatus` change than can unwrap `errors.WithStack`.